tracker/management/commands/import_channels.py

```diff
@@ -0,0 +1,82 @@
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from tracker.models import (
+    Channel,
+    ChannelRevision,
+    Package,
+    PackageAttributeName,
+    PackageName,
+    PackageVersion,
+)
+from tracker.packages import (
+    get_revision_info,
+    list_channel_revisions,
+    list_channels,
+    load_package_list,
+)
+
+
+class Command(BaseCommand):
+    help = "Import Nix channels into the local database"
+
+    def handle(self, *args, **options):
+        for channel in list_channels():
+            if any(
+                channel.name.startswith(x)
+                for x in [
+                    "14.04",
+                    "14.12",
+                    "13.10",
+                    "15.",
+                    "16.",
+                    "17.03",
+                    "17.09",
+                    "18.03",
+                    "18.09",
+                    "19.03",
+                    "19.09",
```

Comment:
Probably can do that. That script is just a huge hack right now. My plan is to give it all of these things as cli arguments. There is no point in hardcoding these.

Also the only reason I excluded these was because they do not have a `packages.json.br` on the S3 bucket. Eventually we should add the logic to create those files based on an arbitrary URL that points to a nixpkgs tarball.

Reply:
Just tried what you recommended in another project and it wouldn't work:
```console
>>> packaging.version.parse("nixos-20.09")
<LegacyVersion('nixos-20.09')>
>>> packaging.version.parse("20.09")
<Version('20.9')>
>>> packaging.version.parse("20.09-unstable")
<LegacyVersion('20.09-unstable')>
```

At the point where we extract the version part (e.g. stripping aways the `nixos-` prefix and arbitrary suffixes) we might as well use a regex...